packages/manager/modules/vrack/src/vrack.controller.js

```diff
@@ -180,7 +180,19 @@ export default class VrackMoveDialogCtrl {
     return this.OvhApiVrack.Aapi()
       .allowedServices({ serviceName: this.serviceName })
       .$promise.then((allServicesParam) => {
-        let allServices = allServicesParam;
+        let allServices = {
+          ...allServicesParam,
+          dedicatedCloud: allServicesParam.dedicatedCloud.filter((service) =>
```

Comment:
nitpicking but you can simply do for your filters :
```js
allServicesParam.dedicatedCloud.filter(VrackMoveDialogCtrl.isServiceAllowed)
```
:)

Reply:
Done, except for the `dedicatedServerInterface` as I must use `dedicatedServerInterface.dedicatedServer`